### PR TITLE
Seed database with pageviews

### PR DESCRIPTION
**Pull request by vinibrsl**
This commit adds basic support for database seeding useful for local testing, especially dashboard changes, like intervals.

It creates two years of pageviews with random timestamps. There is lot of room for improvement, such as adding sources, entry pages, geolocation, devices, custom events, but this already helps us with testing.

Screen Shot 2022-11-16 at 12 56 42

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -71,9 +71,8 @@ defmodule Plausible.MixProject do
       {:dialyxir, "~> 1.0", only: [:dev, :test], runtime: false},
       {:double, "~> 0.8.0", only: :test},
       {:ecto_sql, "~> 3.0"},
-      {:elixir_uuid, "~> 1.2", only: :test},
```
Comment (vinibrsl) [inline mix.exs:74]: This was used in tests, and I replaced with `Ecto.UUID.generate/0` which should be the same.

**Comment by ukutaht** [comment]
Looks very useful for local testing. I think we should also add a note about this in the `CONTRIBUTING.md` document.

File: mix.exs (hunk 1 above)
Comment (aerosol) [reply mix.exs:74]: :fire:

**Review by aerosol — APPROVED** [review]
Awesome!

**Comment by vinibrsl** [comment]
> Looks very useful for local testing. I think we should also add a note about this in the `CONTRIBUTING.md` document.

Good call!

**Review by ukutaht — APPROVED** [review]

**Pull request merged**